components/form/KeyValue.vue

```diff
@@ -431,11 +431,10 @@ export default {
       const text = event.clipboardData.getData('text/plain');
       const lines = text.split('\n');
       const splits = lines.map((line) => {
-        if (line.includes(':')) {
-          return line.split(':');
-        }
+        // eslint-disable-next-line unicorn/prefer-includes
+        const splitter = line.indexOf(':') === -1 || ((line.indexOf('=') < line.indexOf(':')) && line.indexOf(':') !== -1) ? '=' : ':';
```

Comment:
I personally prefer not to disable linting rules wherever possible and I think something like this would be entirely valid

`!line.includes(':') || ((line.indexOf('=') < line.indexOf(':')) && line.includes(':')) ? '=' : ':';`

Reply:
Good point, I seem to remember using `line.includes(':')` earlier on this and the linter still complained at me because of the `((line.indexOf('=') < line.indexOf(':'))` elsewhere in the line. Gonna give this a bit of a test again to see if it still complains but you're right, I only really need need `indexOf` to compare the positions and your suggestion should be functionally the same.